storage/aptosdb/src/state_store/state_snapshot_committer.rs

```diff
@@ -75,7 +75,7 @@ impl StateSnapshotCommitter {
                     let version = delta_to_commit.current_version.expect("Cannot be empty");
                     let base_version = delta_to_commit.base_version;
 
-                    let (batch, root_hash) = self
+                    let (top_levels_batch, sharded_batch, root_hash) = self
```

Comment:
maybe consider having a new struct wrapping top-level batch and sharded_batches, which appear together?

Reply:
It's likely the current implementation is just a temporary solution. I'm considering to make the data sharded at an much earlier stage, to get better parallelism.